samples/Friends/Friends/packages.config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
delete this.

Reply:
Was added by VS 2019 - I've removed the statement.